app/controllers/orga/conferences_controller.rb

```diff
@@ -1,53 +1,58 @@
 class Orga::ConferencesController < Orga::BaseController
+  before_action :find_resource, only: [:show, :edit, :update, :destroy]
```

Comment:
Comes down to personal preference, but I'd prefer this method to be named `find_conference` as it's more tied to what's actually happening. With before filters being a tricky thing in general, I think being as explicit as possible can still be helpful here.